modules/core/src/main/java/org/apache/ignite/mxbean/IgniteMXBean.java

```diff
@@ -388,11 +388,31 @@
     )
     public boolean pingNode(String nodeId);
 
+    /** Activates cluster. */
+    @MXBeanDescription(
```

Comment:
Seems we can make one-line them.

Reply:
Fixed. Also the others.

Reply:
@nizhikov , how do you think, can we keep IgniteMXBean as it is now in this PR? With one new 

`clusterState(String state, boolean force)`

Unfortunately, we can't use two different default-methods in the interfaces IgniteMXBean and Ignite:
`    
default public void active(boolean active){
        active(active, false);
    }

and
default public void active(boolean active){
    active(active, true);
}
`

Jaca compilator forces us to choose one in IgniteKernal:
`class org.apache.ignite.internal.IgniteKernal inherits unrelated defaults for active(boolean) from types org.apache.ignite.Ignite and org.apache.ignite.mxbean.IgniteMXBean`

I think this happens because there are no real implementations of default methods within interfaces. They are just code sample for in-lining to the implementation. So, we have two different implementations.